llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1092,6 +1024,84 @@ void VPlanTransforms::simplifyRecipes(VPlan &Plan, Type &CanonicalIVTy) {
   }
 }
 
+void VPlanTransforms::simplifyBlends(VPlan &Plan) {
```

Comment:
Sorry I missed this earlier, but it looks like simplifyBlends can be made static, and doesn't need to be added in the header?

Reply:
Good point, removed